app/cdap/components/GridTable/utils.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { getCurrentNamespace } from 'services/NamespaceStore';
+import T from 'i18n-react';
+import { IBreadcrumbItemProps } from 'components/GridTable/components/Breadcrumb/types';
+const PREFIX = 'features.WranglerNewUI.Breadcrumb';
+
+export const getWrangleGridBreadcrumbOptions = (workspaceName, location) => {
+  const finalBreadcrumbList: IBreadcrumbItemProps[] = [
+    {
+      link: `/ns/${getCurrentNamespace()}/wrangle`,
+      label: T.translate(`${PREFIX}.labels.wrangleHome`).toString(),
+    },
+  ];
+
+  const requestFromAddress = location?.state?.from;
+  const sourcePath =
```

Comment:
Why are path components being translated?

Reply:
we were actually checking the requestionFromAddress(it is a label) value here. So, we thought of comparing it with the translated value. 

The remaining code has been updated.